src/components/sandbox/ComponentSerializer.ts

```diff
@@ -114,6 +120,23 @@ export class ComponentSerializer {
       return children;
     }
 
+    
+    if ((this.serializer instanceof AngularReactiveSerializer)) {
+      
+      if(children.startsWith('<goa-checkbox')) {
+        // Step 1: Remove goaChecked if both goaChecked and goaValue exist
+        if (children.includes('goaChecked') && children.includes('goaValue')) {
+          children = children.replace(/\bgoaChecked\b\s?/g, '');
+        }
+
+        // Step 2: Replace disabled=true with [attr.disabled]="true"
+        if (children.includes('disabled=true')) {
+          children = children.replace(/\bgoaChecked\b\s?/g, '');
+          children = children.replace(/disabled=true/g, '[attr.disabled]="true"');
```

Comment:
My Angular skills aren't the best, so does `[attr.disabled]` work? I ask because we have never used `attr.` before, nor do the upcoming angular components.